alby/alby_oauth_service.go

```diff
@@ -1166,43 +1166,196 @@ func (svc *albyOAuthService) GetLSPChannelOffer(ctx context.Context) (*LSPChanne
 	return lspChannelOffer, nil
 }
 
+func (svc *albyOAuthService) GetLSPInfo(ctx context.Context, lsp, network string) (*LSPInfo, error) {
```

Comment:
here we call it `lsp` but other places we call it `lspIdentifier`, should we call it `lspIdentifier` here?